CollectionActions.rb

```diff
@@ -7,108 +7,110 @@
 # These actions should involve the WHOLE plate not individual wells.
 # NOTE: The collection is doing the whole action
 module CollectionActions
-  # Store all input collections from all operations
+  # Store all items used in input operations
   #
   # @param operations [OperationList] the list of operations
-  # @param location [String] the location that the items are to be moved to
+  # @param location [String] the storage location
   def store_input_collections(operations, location: nil)
-    show do
-      title 'Put Away the Following Items'
-      table table_of_job_object_location(operations, role: 'input',
-              location: location)
-    end
+    store_collection_materials(operations, location: location, role: 'input')
+    # show do
+    #   title 'Put Away the Following Items'
+    #   table material_storage_locations(operations, role: 'input',
+    #           location: location)
+    # end
   end
 
-  # Stores all output collections from all operations
+  # NOTE: Is there a reason these are seperate methods? 
```

Comment:
Is there a reason these are separate methods? Could just be one with the role specified when you call it.

Reply:
These could be the same method with role called yes